website/src/scripts/header.js

```
@@ -0,0 +1,125 @@
+    burger?.setAttribute('aria-expanded', 'false');
+    burger?.setAttribute('aria-label', 'Open menu');
+    drawer?.setAttribute('aria-hidden', 'true');
+    drawer?.setAttribute('inert', '');
+    burger?.focus();
+  }
+
+  burger?.addEventListener('click', openDrawer);
+  drawerClose?.addEventListener('click', closeDrawer);
+  drawerScrim?.addEventListener('click', closeDrawer);
+
+  document.addEventListener('keydown', e => {
```

ESLint

[error] 'document' is not defined.

(no-undef)